packages/botonic-core/src/models/hubtype-analytics.ts

```diff
@@ -0,0 +1,157 @@
+export enum EventAction {
```

Comment:
so moving them here is just to avoid having analytics as an extra dependency in fbplugin and react right?

Reply:
Fixed: replaced type assertion with errors.As for proper wrapped error handling